docs/General/Manuals/client-manual.rst

```diff
@@ -1111,10 +1111,160 @@ a table below the widget, such as Figure 26, that displays the amount of time un
 Cities View
 ^^^^^^^^^^^
 
+The :guilabel:`Cities View` is a separate page in a table format. When you click the icon for it as shown in
+Figure 27, the client will leave the view you are currently showing and will switch to a listing of your
```

Comment:
Same as before, "button" instead of "icon" and consider rephrasing to just "the client will switch to..."